upup/pkg/fi/cloudup/awstasks/dnsname.go

```diff
@@ -163,6 +204,14 @@ func (_ *DNSName) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *DNSName) error
 		}
 	}
 
+	if e.TargetNetworkLoadBalancer != nil {
+		rrs.AliasTarget = &route53.AliasTarget{
+			DNSName:              e.TargetNetworkLoadBalancer.DNSName,
+			EvaluateTargetHealth: aws.Bool(false),
+			HostedZoneId:         e.TargetNetworkLoadBalancer.HostedZoneId,
+		}
+	}
+
```

Comment:
Will eventually need corresponding changes to `RenderCloudformation` and `RenderTerraform`. We should at least make sure it refuses to allow NLB with Cloudformation or Terraform rather than generating a non-working cluster.

Reply:
I implemented this by panic via the following methods.
NetworkLoadBalancer.TerraformLink() 
NetworkLoadBalancer.CloudformationAttrCanonicalHostedZoneNameID() NetworkLoadBalancer.CloudformationAttrDNSName()